app/src/main/java/fr/free/nrw/commons/contributions/ContributionBoundaryCallback.kt

```diff
@@ -27,7 +27,8 @@ class ContributionBoundaryCallback @Inject constructor(
      * network
      */
     override fun onZeroItemsLoaded() {
-        fetchContributions()
+        if (sessionManager.userName != null)
```

Comment:
Could you please move this to the first line of `fetchContributions() {`?
Also, please add brackets {} around, because even if there is only one instruction inside, that's more future-proof.
Thanks! :-)